uPortal-web/src/main/java/org/apereo/portal/portlet/dao/jpa/MarketplaceRatingImpl.java

```diff
@@ -33,6 +34,9 @@
     @Column(name = "REVIEW", length = REVIEW_MAX_LENGTH)
     private String review;
 
+    @Column(name = "RATINGDATE")
+    private Date ratingDate;
```

Comment:
This is a great addition, but (for a few reasons) we need to proceed with care.

One of those reasons is that this enhancement includes a database schema change.